sqrl-planner/sqrl-common/patch.diff

```diff
@@ -0,0 +1,546 @@
+1033,1044c1033,1045
```

Comment:
Is this copied from somewhere? What is unique to SQRL?

Reply:
Not copied. This is what needs to be changed in the parser to loosen some rules and add the join syntax.